src/pcv/PCVRouter.sol

```diff
@@ -98,7 +101,12 @@ contract PCVRouter is IPCVRouter, CoreRefV2 {
         uint256 amount,
         address sourceAsset,
         address destinationAsset
-    ) external whenNotPaused onlyVoltRole(VoltRoles.PCV_MOVER) globalLock(1) {
```

Comment:
We cannot remove the globalLock to level 1 on the unchecked because both withdraw and deposit require being at lock level 1 before locking up to level 2.

Reply:
ha good point